qa/arranger/deployments.yml

```diff
@@ -0,0 +1,82 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    fluxcd.io/automated: 'true'
+    fluxcd.io/tag.copy-zonefile: semver:~3.13.6
```

Comment:
^ This is not needed, given that clin is not behind an ssh tunnel.

Reply:
the whole annotations section?

Reply:
No, just those two parts:

```
fluxcd.io/tag.copy-zonefile: semver:~3.13.6
fluxcd.io/tag.coredns: semver:~1.8.4
```

The flux annotation to auto-deploy arranger is still required.

Reply:
So, you would leave this part:

```
fluxcd.io/automated: 'true'
```

Given that a filter for arranger is not specified, it should just take the latest docker image in the registry